### ISSUE TITLE
When Android 15?

### ISSUE THREAD

**Issue by mytja** [mytja/treble_evo#58]
I am aware that Evolution X will likely release Android 15 (Evolution X 10.0?) in the near future (November). However that doesn't mean that you're free to open issues asking for Android 15. I will release it once I think it meets my quality standards. Before I release any Android 15 build, I'll finish everything off with a final stable Android 14 build (hopefully a November patch).

A lot of work (TrebleDroid team's, mine and other developers') has gone into making it quite stable from my experience. I'd like to thank [phhusson](https://github.com/phhusson), [ponces](https://github.com/ponces), [AndyCGYan](https://github.com/AndyCGYan) and many others for their work on GSIs.

And as for initial release, a lot of work needs to be done from my side to port patches over to a new version. I'll redo the entire set of TrebleDroid patches and will make sure none will be missed. It's a lot of work and if the patches don't result in a satisfying (with as few bugs and missing features as possible) system, I won't release A15 until everything will be polished. Ponces' AOSP15 has quite a few bugs from my experience (but I don't blame him for it, it's of course still pre-release), so I'm not that positive about current polishment status.

Once I release the first A15 build, I'll clean up open issues. IF YOUR DEVICE WON'T BOOT ON ANDROID 15, SEND THE GODDAMN LOGS. OR NOT. From now on I reserve the right to close any issue not providing the log and will do so. I also won't be fixing any/most Bluetooth bugs, as that's likely issue with your vendor.

If everything goes according to plan (which it won't), you can expect a first build somewhere around 18th November.

**Comment by pixelextend** [comment]
> I am aware that Evolution X will likely release Android 15 (Evolution X 10.0?) in the near future (November). However that doesn't mean that you're free to open issues asking for Android 15. I will release it once I think it meets my quality standards. Before I release any Android 15 build, I'll finish everything off with a final stable Android 14 build (hopefully a November patch).
> 
> A lot of work (TrebleDroid team's, mine and other developers') has gone into making it quite stable from my experience. I'd like to thank [phhusson](https://github.com/phhusson), [ponces](https://github.com/ponces), [AndyCGYan](https://github.com/AndyCGYan) and many others for their work on GSIs.
> 
> And as for initial release, a lot of work needs to be done from my side to port patches over to a new version. I'll redo the entire set of TrebleDroid patches and will make sure none will be missed. It's a lot of work and if the patches don't result in a satisfying (with as few bugs and missing features as possible) system, I won't release A15 until everything will be polished. Ponces' AOSP15 has quite a few bugs from my experience (but I don't blame him for it, it's of course still pre-release), so I'm not that positive about current polishment status.
> 
> Once I release the first A15 build, I'll clean up open issues. IF YOUR DEVICE WON'T BOOT ON ANDROID 15, SEND THE GODDAMN LOGS. OR NOT. From now on I reserve the right to close any issue not providing the log and will do so. I also won't be fixing any/most Bluetooth bugs, as that's likely issue with your vendor.
> 
> If everything goes according to plan (which it won't), you can expect a first build somewhere around 18th November.

Its a Good News Android 15 coming 😉

**Comment by lengge86** [comment]
Thanks. I just moved from another GSI ROM to you rom. It is surprisingly good.

**Comment by TanvirOnGH** [comment]
Same here! Excited for A15 builds.

**Comment by mytja** [comment]
Yeah... about that. I've just built some test images and... it's disappointing to say the least. Fingerprint is not working on my device (I'm currently trying to resolve it), many apps aren't included anymore, Evolver is almost completely blank. I'm seriously thinking of sticking to A14, as these details will be polished over time.

Just as a PSA so that you guys aren't disappointed when (if) I release an A15 pre-release. I understand you guys and I'm seriously thinking of extending A14 support until A15 is fully ready. Evolution X team will in any case maintain A14 for the foreseeable future.

**Comment by Vad1mCSO2** [comment]
yep is coming as Evolut10n X

**Comment by mytja** [comment]
First and only build for this year has just been released. Enjoy, but I don't really know what you can enjoy.

https://github.com/mytja/treble_evo/releases/tag/20241119

**Comment by mytja** [comment]
Android 15 is now fully out

https://github.com/mytja/treble_evo/releases/tag/20250118
